shared/crypto/operations/encrypt/index.tsx

```diff
@@ -40,10 +40,7 @@ type EncryptOptionsProps = {
   options: Types.EncryptOptions
 }
 
-// We want to debuonce the onChangeText callback for our input so we are not sending an RPC on every keystroke
-const debounced = debounce((fn, ...args) => fn(...args), 100)
```

Comment:
we want redux to handle this debouncing. we want to know when things are out of sync immediately. we debounce making the rpc call separately